app/src/main/java/chat/rocket/android/model/params/PushUpdate.java

```diff
@@ -0,0 +1,34 @@
+package chat.rocket.android.model.params;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class PushUpdate {
```

Comment:
This class is not related to Realm model. 
It is better to define this logic as a private method in the subclass of MethodCallHelper.